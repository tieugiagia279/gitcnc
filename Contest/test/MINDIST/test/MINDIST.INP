4
0 1 2 1
-1000000 0 1000000 0
0 12345 10 12345
0 -12345 10 12345
