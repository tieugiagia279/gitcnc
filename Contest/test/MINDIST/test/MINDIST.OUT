2
2000000
24690
24690